standalone-metastore/metastore-server/src/main/resources/package.jdo

```diff
@@ -384,10 +384,13 @@
       <field name="id" primary-key="true" value-strategy="native">
         <column name="CD_ID" jdbc-type="BIGINT" />
       </field>
-      <field name="columns" mapped-by="cd">
+      <field name="fields" mapped-by="cd">
         <order column="INTEGER_IDX"/>
         <collection element-type="MColumn" dependent-element="true" />
       </field>
+      <field name="columns" persistence-modifier="none">
```

Comment:
@saihemanth-cloudera, why do we need this?

Reply:
All the fields declared in JDO are persistent by default. i.e.., Data Nucleus assumes that there is mapping of this field to a column in back end db (here it assumes that there is 1-1 mapping of columns field to a column in CDS table). It explicitly say that this field is only used in application layer and that there is no such column in the DB, we need to declare it as persistence-modifier="none".
`@NotPersistant` annotation mentioned in Java is not taking into effect unless we mention it in `.jdo` class. So I left the annotation for future reference to the developers.

Reply:
without `default-fetch-group="false"` DataNucleus tries to read this field and it fails, so I had to include it.

Reply:
weird, see https://www.datanucleus.org/products/accessplatform_4_1/jdo/fields_properties.html

Reply:
@deniskuzZ  @okumin nit: better to check the doc https://www.datanucleus.org/products/accessplatform_6_0/ i.e 6_0 instead of 4_1 which is for datanucleus 4.x. Hoping the most of the things are same in docs, haven't verified myself :-)

Reply:
same thing
https://www.datanucleus.org/products/accessplatform_6_0/jdo/mapping.html

> Making a field/property non-persistent

If you have a field/property that you don’t want to persist, just mark it’s persistence-modifier as none, like this

````
<class name="mydomain.MyClass">
    <field name="unimportantField" persistence-modifier="none"/>
</class>
````

Reply:
tested and verified. It works.